src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -927,7 +927,7 @@ private bool ShouldContinueDispose(string name, DateTime? wakeup, out int dueTim
                 timer.Dispose();
             }
 
-            if (wakeup.HasValue == false || wakeup.Value == DateTime.MaxValue)
+            if (wakeup.HasValue == false || wakeup.Value == DateTime.MaxValue.ToUniversalTime())
```

Comment:
We can remove this:
`wakeup.Value == DateTime.MaxValue.ToUniversalTime()`

Reply:
This can actually throw, I think.
If you have a local time where the UTC is beyond the limits, no?

Reply:
done